ext/mbstring/tests/mb_trim.phpt

```diff
@@ -0,0 +1,62 @@
+--TEST--
+mb_trim() function tests
+--EXTENSIONS--
+mbstring
+--FILE--
+<?php
+mb_internal_encoding("UTF-8");
+
+echo "== Single string ==\n";
```

Comment:
You meant "Single character". Please add test for `mb_ltrim(' test ', '')`

Reply:
Thank you, your code is crashed. I add test case.